src/command/install/plugin.rs

```diff
@@ -47,10 +61,18 @@ impl Plugin {
             no_prebuilt_binaries.set_suggestion(RoverErrorSuggestion::CheckGnuVersion);
             return Err(no_prebuilt_binaries);
         }
-
-        match (consts::OS, consts::ARCH) {
+        match (os, arch) {
             ("windows", _) => Ok("x86_64-pc-windows-msvc"),
-            ("macos", _) => Ok("x86_64-apple-darwin"),
+            ("macos", _) => {
+                match self {
+                    Self::Router(RouterVersion::Exact(v)) if v.lt(&AARCH_OSX_FIRST_ROUTER_VERSION) => {
```

Comment:
```suggestion
                    Self::Router(RouterVersion::Exact(v)) if v < &AARCH_OSX_FIRST_ROUTER_VERSION => {
```
can we just use the operator here? nitpick ofcourse but that's what it does for linux a few lines below.

Reply:
Oh that makes sense indeed!

Reply:
Fixed in a41eefec